### ISSUE TITLE
OSX shell directory is appearing twice in Integrated Terminal

### ISSUE THREAD

**Issue by vikas027** [Microsoft/vscode#28414]
### Versions
- VSCode Version: 1.13.0
- OS X Version: 10.11.2

### Problem
Current working directory is appearing twice in after upgrading to 1.13.0. I have deleted all user and workspace settings, but the problem still persists.
![ezgif-3-7c73bce357](https://user-images.githubusercontent.com/6998650/27001915-281d9f4e-4e18-11e7-8b63-eaeaa672d0f1.gif)

### Steps to Reproduce
1. Install / Upgrade Visual Code version 1.13.0 on OSX 10.11.2.
2. `View` -> `Integrated Terminal`

### Workaround
Interestingly, [fish](https://fishshell.com) shell is not experiencing this problem. I have switched temporarily to fish 
```$ brew install fish```

**Comment by AaronSum** [comment]
Same issue

**Comment by mlieberman85** [comment]
I am running into this issue under OS X using zsh

**Comment by hawrre** [comment]
Same issue after upgrading to 1.13.0

**Comment by serge-io** [comment]
Working fine on macOS Sierra 10.12.5 (shell directory appears only once as it should)

- macOS Sierra 10.12.5.
- VSCode 1.13.0

**Comment by ramya-rao-a** [comment]
Duplicate of #28328

cc @Tyriar

**Comment by vikas027** [comment]
Duplicate of https://github.com/Microsoft/vscode/issues/26038, https://github.com/Microsoft/vscode/issues/28453, https://github.com/Microsoft/vscode/issues/28304, https://github.com/Microsoft/vscode/issues/28437, https://github.com/Microsoft/vscode/issues/28451
